src/test/java/edu/project1/DictionaryClassTest.java

```diff
@@ -0,0 +1,40 @@
+package edu.project1;
+
+import org.junit.jupiter.api.BeforeEach;
+import org.junit.jupiter.api.DisplayName;
+import org.junit.jupiter.api.Test;
+import static org.assertj.core.api.AssertionsForClassTypes.assertThat;
+import java.util.ArrayList;
+import java.util.List;
+
+public class DictionaryClassTest {
+    Dictionary dictionary;
+    @BeforeEach
+    public void createObject()
+    {
+        List<String> words = new ArrayList<>();
+        words.add("audi");
+        words.add("bmw");
+        words.add("skoda");
+        words.add("lada");
+        dictionary = new Dictionary(words);
+    }
+    @Test
+    @DisplayName("Check that random word is from list")
+    public void checkThatRandomWordIsAtList()
+    {
+        boolean expected = true;
+        boolean response = dictionary.getWords().contains(dictionary.randomWord());
+        assertThat(expected).isEqualTo(response);
```

Comment:
`BUG `Наоборот же
```suggestion
        assertThat(response).isEqualTo(expected);
```
А вообще используй
 assertThat(response).isTrue();

Reply:
Все пофиксил